src/layers/src/icon-layer/icon-info-modal.tsx

```diff
@@ -21,9 +21,8 @@
 import React from 'react';
 import styled from 'styled-components';
 import {line} from 'd3-shape';
-import {CenterFlexbox} from 'components/common/styled-components';
 import {FormattedMessage} from '@kepler.gl/localization';
-import {Table} from 'components/common/icons';
+import {CenterFlexbox, Table} from '@kepler.gl/constants';
```

Comment:
instead of moving `CenterFlexbox` to constant, can you just copy the implementation here. so we don't need to have layers depends on `CenterFlexbox`?